packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1313,21 +1331,47 @@ export default class Meeting extends StatelessWebexPlugin {
    * @memberof Meeting
    */
   setUpLocusInfoSelfListener() {
-    this.locusInfo.on(LOCUSINFO.EVENTS.SELF_MUTED_BY_OTHERS, (payload) => {
-      if (payload) {
+    this.locusInfo.on(LOCUSINFO.EVENTS.SELF_LOCAL_UNMUTE_REQUIRED, () => {
+      if (this.audio) {
+        this.audio.handleServerLocalUnmuteRequired();
```

Comment:
yes, we are unmuting here without user or developer consent. I'm not sure if it's a good idea to get the developer involved here. It will only make us write more code and then you have to document it and you're putting new requirements on the developer so then they have to implement it and if they forget or don't do it because they didn't realise they needed to or don't have time to do it, then the feature won't work and the end-user experience will be bad.